model/task_start_estimation.go

```diff
@@ -0,0 +1,150 @@
+package model
+
+import (
+	"sort"
+	"time"
+
+	"github.com/evergreen-ci/evergreen"
+	"github.com/evergreen-ci/evergreen/model/host"
+	"github.com/evergreen-ci/evergreen/model/task"
+	"github.com/evergreen-ci/evergreen/util"
+	"github.com/mongodb/grip"
+	"github.com/mongodb/grip/message"
+	"github.com/pkg/errors"
+)
+
+const (
+	HostInitializingDelay  = 4 * time.Minute
+	HostStartingDelay      = 3 * time.Minute
+	HostProvisiongingDelay = 1 * time.Minute
```

Comment:
don't export these values. 

also comment explaining that these are estimates based on what we observe from aws and not something intrinsic to the system.

Reply:
done